### PR TITLE
Center sponsor image

### LINKED ISSUE #179: Sponsor logos are left-aligned
image

They should be in the center of their cell

### PR DESCRIPTION
**Pull request by ElenaExposito**
# The Pull Request is ready

- [x] fixes #179
- [x] all actions are passing
- [x] only fixes a single issue

## Website

- [x] mobile view is useable
- [x] desktop view is useable
- [x] no a-tags are used direktly (NavLink, MailLink, ExternalLink instead)
- [ ] all new texts are added to the translation files (at least the english one)
- [ ] tests have been added (if required)
- [ ] shared code has been extracted in a different file

### CODE REVIEW

File: website/src/index.css [hunk 1]
```diff
@@ -210,6 +210,12 @@ footer li:last-child::after {
   grid-template-columns: 1fr;
 }
 
+.sponsor-image {
```
Comment (Idrinth) [inline website/src/index.css:213]: wouldn't an `align-self` and `justify-self` on the `.sponsor img` be simpler? Would also remove the need for the div.

File: website/src/pages/contributing/sponsors/index.tsx [hunk 2]
```diff
@@ -18,10 +18,12 @@ const Index = () => <Layout
...
     </div>
     <div className={'card sponsor'}>
-      <img
-        src={bjoernbuettner}
-        alt={'Bjoern Büttner\'s Backing'}
-      />
+      <div className={'sponsor-image'}>
+        <img
+          src={bjoernbuettner}
+          alt={'Bjoern Büttner\'s Backing'}
```
Comment (Idrinth) [inline website/src/pages/contributing/sponsors/index.tsx:24]: not quite sure why there is an `oe`, could you replace it with `ö`?

**Review by Idrinth — CHANGES_REQUESTED** [review]
Thanks a lot for helping! I think you might be able to achieve the same with a bit less code

File: website/src/index.css (hunk 1 above)
Comment (ElenaExposito) [reply website/src/index.css]: You're right! Changes pushed

File: website/src/pages/contributing/sponsors/index.tsx (hunk 2 above)
Comment (ElenaExposito) [reply website/src/pages/contributing/sponsors/index.tsx]: Sure! Already pushed

**Review by Idrinth — APPROVED** [review]

**Pull request merged**